chemprop/models/model.py

```diff
@@ -49,7 +49,16 @@ def create_encoder(self, args: TrainArgs) -> None:
         :param args: A :class:`~chemprop.args.TrainArgs` object containing model arguments.
         """
         self.encoder = MPN(args)
-
+              
+        if hasattr(args,'checkpoint_frzn'):
```

Comment:
Even if checkpoint_frzn hasn't been provided, it will still have a default value of None and will still trigger this if statement.